### DISCUSSION TITLE
Cache views

### DISCUSSION THREAD

**Discussion by rogervila** [inertiajs/inertia#130] [Ideas]
Hello,

I have some inertia views that serve static content (ie: "about" or "terms & conditions" page)

It might be interesting to add the ability to cache this kind of views using `localStorage`, preventing inertia to hit the server.

This would be an example:

```php
// Inertia will cache the response on localStorage during 300 seconds
Inertia::render('About')->cache(300);
```

What do you think?

**Comment by joydeep-bhowmik** [comment]
Nope!! They are not going to add it I suppose

**Comment by rogervila** [comment]
@reinink would you accept a PR for that feature?
